src/utils/io/progress_log.rs

```diff
@@ -233,14 +235,14 @@ mod tests {
         progress.last_logged_items = 1024 * 1024 * 1024 / 3;
         assert_eq!(
             progress.msg(now + Duration::from_secs(125)),
-            "test 512 MiB / 1 GiB, 50%, 170.7 MiB/s, elapsed time: 2m 5s"
+            "test 512 MiB / 1 GiB, 50%, 1.4 MiB/s, elapsed time: 2m 5s"
```

Comment:
Could you please explain why the speed is 1.4 MiB/s? It'd be nice to have a comment with the math regarding the expectations.

Reply:
That's a good point.

Reply:
Thanks for adding comments, that helps.